src/rust/rust-proto-new/Cargo.toml

```diff
@@ -12,17 +12,16 @@ futures = "0.3"
 prost = "0.10"
 prost-types = "0.10"
 thiserror = "1.0"
-tokio = { version = "1.14", features = ["rt", "time"] }
+tokio = { version = "1.17", features = ["rt", "time"] }
```

Comment:
We probably want to use the same tokio features everywhere

Reply:
why?